force-app/main/default/classes/OpportunityHelper.cls

```diff
@@ -0,0 +1,146 @@
+public with sharing class OpportunityHelper {
+    
+    public static void amountValidation (List<Opportunity> newOpps) {
+        for(Opportunity opp : newOpps){
+            if(opp.Amount < 5000){
+                opp.addError('Opportunity amount must be greater than 5000');
```

Comment:
It's a good practice to move values like these to variable so they are easier to maintain